src/librustc_mir/util/borrowck_errors.rs

```diff
@@ -356,12 +356,14 @@ pub trait BorrowckErrors<'cx>: Sized + Copy {
                                  use_span: Span,
                                  verb: &str,
                                  optional_adverb_for_moved: &str,
-                                 moved_path: &str,
+                                 moved_path: Option<String>,
                                  o: Origin)
                                  -> DiagnosticBuilder<'cx>
     {
+        let moved_path = moved_path.map(|mp| format!(": `{}`", mp)).unwrap_or("".to_owned());
```

Comment:
do we have a test that shows the effect of this change? I don't see one... or is it that other tests regressed as a result?

Reply:
the stderr files changed like in https://github.com/rust-lang/rust/pull/51688/files#diff-4d3cc9be8c38f963314cf51e57d3fcbcL23